netket/operator/_local_liouvillian.py

```diff
@@ -244,7 +244,7 @@ def get_conn_flattened(self, x, sections, pad=False):
                 max_conns_Lrc += max_lr * max_lc
 
         # compose everything again
-        if self._xprime.shape[0] < self._max_conn_size * batch_size:
+        if self._xprime_f.shape[0] < self._max_conn_size * batch_size:
```

Comment:
I don't quite understand why this makes a difference. Aren't both `_xprime` and `_xprime_f` pretty much the same type of array and class attributes?

Reply:
Yes but this is reshaping _f which is much bigger. So it was checking that the size of not _f was sufficiently large (never true) and then resizing _f, so that check was always false